tuna/miopen_tables.py

```diff
@@ -460,42 +461,37 @@ def solver_id(self):
                   ForeignKey("solver.id",
                              onupdate="CASCADE",
                              ondelete="CASCADE"),
-                  nullable=False)
+                  nullable=True)
 
-  golden_miopen_v = Column(String(length=64), nullable=False)
-  is_winning = Column(TINYINT(1), nullable=False, server_default="0")
-  is_app = Column(TINYINT(1), nullable=False, server_default="0")
-  arch = Column(String(length=20), nullable=False, server_default="")
-  num_cu = Column(Integer, nullable=False, server_default="0")
+  golden_miopen_v = Column(Integer, nullable=False)
 
 
 class ConvolutionGolden(BASE, GoldenMixin):
   """Golden table for convolution"""
   __tablename__ = "conv_golden"
   __table_args__ = (UniqueConstraint("golden_miopen_v",
-                                     "find_db",
+                                     "fdb_key",
                                      "config",
                                      "session",
```

Comment:
Shouldn't be unique on session. Would allow arch/cu/config/solver from separate sessions to have the same golden_v.

Reply:
Can change this unique to golden_v/arch/cu/config/solver. config == fdb_key